4 Tests passed.

kayanr opened the pull request:
Please review Javascript for missingNumber(numbers) and removeDuplicates(names) functions. 4 Tests passed so far. Still working on the previous functions.

kayanr committed:
added pseudocode and code to missingNumber(numbers) function. 2 Tests passed.

Added pseudocode and code  to missingNumber(numbers) function. 2 Tests passed.

kayanr committed:
Added pseudocode and code to removeDuplicates(names) function. 4 Tests

Added pseudocode and code to removeDuplicates(names) function. 4 Tests.

kayanr committed:
Added pseudocode and code to averageAge(ages) function. 6 Tests passed.

Added pseudocode and code to averageAge(ages) function. 6 Tests passed,

kburd commented on QuizDifferently.js:
```diff
@@ -1,67 +1,101 @@
+
+    //return names from no duplicate name roster, "noDupsRoster"
+    return noDupsRoster;
+  }
```
Nice solution! I challenge you to use the `foreach` iteration method here as well

kburd commented on QuizDifferently.js:
```diff
@@ -1,67 +1,101 @@
+    let missingNum = 0;
+
+    //sum numbers from 1 to 100
+    for (let i = 1; i <= 100; i++) {
```
There is a formula to calculate the sum from 1 to n. Take a look and try to implement it here. I for loop works but the formula works in constant time

kayanr replied:
Thanks for your feedback Kaleb. I appreciate it. I will implement your suggestions.

kburd reviewed (commented):
Awesome work so far!

kayanr committed:
updated removeDuplicates(names)  function to use forEach loop  instead of  for loop

updated removeDuplicates(names)  function to use forEach loop instead of  for loop. This section inserts roster of names to a Set object.

kayanr committed:
updated how to sum numbers from 1 to 100

updated how to sum numbers from 1 to 100
